myeongseok/app.js

```diff
@@ -43,41 +48,45 @@ app.post('/users/signup', (req, res, next) => {
     `,
     [name, email, profile_image, password]
   );
+
   res.status(201).json({ message: 'userCreated' });
 });
 
 app.post('/posts', (req, res, next) => {
-  const { title, content, user_id } = req.body;
+  const { title, postImageUrl, content, user_id } = req.body;
```

Comment:
```suggestion
  const { title, postImageUrl, content, userId } = req.body;
```
snake_case가 아닌 camelCase로 작성하고 받을 수 있게 해주세요~